dashboard/app/views/shared/_student_continue.html.haml

```diff
@@ -12,6 +12,6 @@
     #{t('puzzle')} #{script_level.position}
   &nbsp;
   = link_to build_script_level_url(script_level) do
-    %button.btn.btn-primary.button-continue#continue_button= t('continue')
+    %button.btn.btn-primary.button-continue#continue_button{class: role_class}= t('continue')
```

Comment:
looks like there's a test failure because you're not passing in role_class when this partial is rendered from dashboard's homepage